domain/src/main/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/stønadsperiode/MaskinellAldersvurderingMedGrunnlagsdata.kt

```diff
@@ -1,14 +1,17 @@
 package no.nav.su.se.bakover.domain.søknadsbehandling.stønadsperiode
 
 import no.nav.su.se.bakover.common.domain.Stønadsperiode
+import no.nav.su.se.bakover.common.domain.sak.Sakstype
+import no.nav.su.se.bakover.common.domain.tid.isEqualOrBefore
 import no.nav.su.se.bakover.common.domain.tid.startOfMonth
 import person.domain.Person
+import vilkår.uføre.domain.ALDER_MINSTE_ALDER
 import vilkår.uføre.domain.UFØRETRYGD_MAX_ALDER
 import java.time.LocalDate
 import java.time.Year
 
 /**
- * Brukes per nå bare for SU-uføre. Kan utvides videre når SU-Alder skal inn
+ * Brukes per nå bare for SU-uføre og SU-alder.
```

Comment:
```suggestion
 * Brukes både for SU-uføre og SU-alder.
```

Kan vurdere og bare slette denne typen kommentarer 🤷